src/pages/starter-kit/checkout/getting-started.md

```diff
@@ -27,57 +27,6 @@ You must install or have access to the following prerequisites to develop with t
 - Access to the [Adobe Developer Console](https://console.adobe.io/) with an App
   Builder license. If you do not have access to the Adobe Developer Console or App Builder, refer to [get access to App Builder](https://developer.adobe.com/app-builder/docs/overview/getting_access/#get-access-to-app-builder).
 
-## Install Commerce modules
-
-<InlineAlert variant="info" slots="text"/>
-
-Adobe Commerce as a Cloud Service is preconfigured with all the required modules for the checkout starter kit. Cloud Service users can proceed by [configuring their local environment](#initial-configuration) or [configuring Commerce](./configure.md).
-
-<Edition name="paas" />
-
-Before installing Commerce modules, ensure that you have the required credentials in `auth.json` with [access to the Adobe Commerce repository](https://experienceleague.adobe.com/en/docs/commerce-operations/installation-guide/prerequisites/authentication-keys).
-
-### Install the Out-of-Process Payment module
-
-To enable out-of-process payment methods in Commerce, install the `magento/module-out-of-process-payment-methods` module. This module enables out-of-process payment functionalities.
-
-To install the module, run the following command using Composer:
-
-```bash
-  composer require magento/module-out-of-process-payment-methods --with-dependencies
-```
-
-### Install the Out-of-Process Shipping module
-
-To enable out-of-process shipping methods in Adobe Commerce, install the `magento/module-out-of-process-shipping-methods` module.
-To install the module, run the following command using Composer:
-
-```bash
-  composer require magento/module-out-of-process-shipping-methods --with-dependencies
-```
-
-### Install the Out-of-Process Tax module
-
-To enable out-of-process tax management in Adobe Commerce, install the `magento/module-out-of-process-tax-management` module.
-To install the module, run the following command using Composer:
-
-```bash
- composer require magento/module-out-of-process-tax-management --with-dependencies
-```
-
-#### Out-of-Process Tax Management Limitations
-
-This extension overrides the class `Magento\Tax\Model\Sales\Total\Quote\Tax` in the `di.xml` file
-
-```xml
-<preference for="Magento\Tax\Model\Sales\Total\Quote\Tax"
-                type="Magento\OutOfProcessTaxManagement\Model\Tax\Sales\Total\Quote\Tax"/>
-```
-
-As a result, this extension is not compatible with other tax extensions that also override this class, such as TaxJar or Avalara.
-
-If you need to use another tax extension, Adobe recommends disabling this extension to prevent conflicts.
-
 ### Install the Commerce Eventing module (Commerce 2.4.4 and 2.4.5 only)
 
```

Comment:
Shouldn't we keep "## Install Commerce modules" section, but mention that each module is located under each domain page, as have done in https://adobedocs.github.io/commerce-extensibility/starter-kit/checkout/use-cases/?

<img width="400" alt="Screenshot 2025-05-08 at 15 00 58" src="https://github.com/user-attachments/assets/0fec723a-11df-4dde-a233-f33bb8f9e8b2" />